### PR TITLE
feat(config): support starting pd nodes & store nodes

### PR DESCRIPTION
**Pull request by HJ-Young**
subtask of https://github.com/apache/incubator-hugegraph/issues/2575

### CODE REVIEW

File: src/common/deploy_graph.py
Comment (imbajin) [inline src/common/deploy_graph.py:1]: could use 100 or 120 as the line wrapping standard, no need wrap too early

File: README.md [hunk 1]
```diff
@@ -23,7 +23,12 @@ Note: modify the configs in basic_config.py before run the test script
 2. modify server/toolchain_git_branch_commit
 3. modify auth/https/serice ports if need..
 '''
-python deploy_start.py all # or 'server' or 'toolchain'
+python deploy_start.py all # or 'server' or 'toolchain' or 'hugegraph'
```
Comment (imbajin) [inline README.md:26]: hugegraph represents "server + pd + store"?

File: src/deploy_start.py [hunk 2]
```diff
@@ -32,8 +38,13 @@ def graph_deploy(param, conf_obj):
...
+            and sys.argv[1] in ['all', 'server', 'toolchain', 'pd', 'store', 'hugegraph']:
+        graph_deploy(sys.argv[1], basic_config)
+    elif param_size == 3 \
+            and sys.argv[1] in ['all', 'server', 'toolchain', 'pd', 'store', 'hugegraph']\
+            and sys.argv[2] in ['hbase', 'hstore', 'cassandra', 'mysql', 'rocksdb', 'scylladb']:
+        basic_config.server_backend = sys.argv[2]
         graph_deploy(sys.argv[1], basic_config)
     else:
-        print('failed: 执行脚本参数为[all,server,toolchain]')
+        print('failed: 执行脚本参数为[all,server,toolchain, pd, store, hugegraph]')
```
Comment (imbajin) [inline src/deploy_start.py:49]: sort the order from big -> small (scope)

File: README.md (hunk 1 above)
Comment (HJ-Young) [reply README.md:26]: yes, use hugegraph to start a cluster including 1 pd, 1 store and 1 server

File: src/deploy_start.py (hunk 2 above)
Comment (imbajin) [inline src/deploy_start.py:49]:
```suggestion
        print('failed: 执行脚本参数为[all, hugegraph, toolchain, server, pd, store]')
```

**Review by imbajin — APPROVED** [review]

**Pull request merged**